runtime/doc/health.txt

```diff
@@ -21,6 +21,17 @@ To run all healthchecks, use: >vim
 <
 Plugin authors are encouraged to write new healthchecks. |health-dev|
 
+                                                                    *g:health*
+g:health  This global variable controls the behavior and appearance of the
+          `health` floating window. It should be a dictionary containing the
+          following optional keys:
+          - `style`: string? Determines the display style of the `health` window.
```

Comment:
This is a temporary hack until we have `:help preview-popup` or possibly a `:float` modifier. https://github.com/neovim/neovim/issues/31902